acc_test/data/invalid_kafka_user_create_duplicate.tf

```diff
@@ -0,0 +1,60 @@
+// This is part of testing "kafka user" suite, 5 of 5 
```

Comment:
adding a new config for testing will add a fair chunk of time to our acceptance tests (considering that this is kafka user test, meaning that it requires RUNNING cluster). I suggest plugging in testing the duplicate user creation in one of the other tests